CI

```
Error: D:\a\docs\docs\samples\snippets\csharp\VS_Snippets_Misc\tpl_cancellation\cs\cancel1.cs(64,31): error CS1061: 'CancellationTokenSource' does not contain a definition for 'CancelAsync' and no accessible extension method 'CancelAsync' accepting a first argument of type 'CancellationTokenSource' could be found (are you missing a using directive or an assembly reference?) [D:\a\docs\docs\samples\snippets\csharp\VS_Snippets_Misc\tpl_cancellation\cs\Project.csproj::TargetFramework=net6.0]
```

So I guess it should remain Cancel.

--- a/samples/snippets/csharp/VS_Snippets_Misc/tpl_cancellation/cs/cancel1.cs
+++ b/samples/snippets/csharp/VS_Snippets_Misc/tpl_cancellation/cs/cancel1.cs
@@ -61,7 +61,7 @@ public class Example
         char ch = Console.ReadKey().KeyChar;
         if (ch == 'c' || ch == 'C')
         {
-            await tokenSource.CancelAsync();
+            tokenSource.Cancel();
             Console.WriteLine("\nMain: Task cancellation requested.");
 
             // Optional: Observe the change in the Status property on the task.

